packages/mssql/src/Adaptor.js

```diff
@@ -37,7 +36,7 @@ function createConnection(state) {
       port,
       database,
       encrypt,
-      rowCollectionOnRequestCompletion,
+      rowCollectionOnRequestCompletion: true,
```

Comment:
@mtuchi  question: if we have it here, does it mean a user cannot chose to make it a false?

Reply:
Yes that's correct. The current implementation of `new Request(query, (err, rowCount, rows)` depends on that option to be `true` for rows to be returned. That's why it's not a configuration option rather an internal option that needs to always be `true`

@hunterachieng